src/Microsoft.Dnx.Host.CoreClr/DomainManager.cs

```diff
@@ -11,8 +12,24 @@
 [SecurityCritical]
 sealed class DomainManager
 {
+    [StructLayout(LayoutKind.Sequential)]
+    private unsafe struct NativeBootstrapperContext
+    {
+        public char* OperatingSystem;
```

Comment:
Having the version number be formatted like a System.Version mostly removes my concern here though. The version number format "81" was a RID-specific concept so if that was leaking in I figured the whole thing should. Since it's not, I'm OK leaving this as-is